site/en/tutorials/text/word2vec.ipynb

```diff
@@ -959,7 +959,7 @@
         "  return tf.squeeze(vectorize_layer(text))\n",
```

Comment:
I think `vectorize_text` is just left over from an earlier version, before the layer could handle this particular input. 

So I'm +1 on just deleting the function and using the layer.

Reply:
Done! Thanks :smile: